File: Backend/Backend/Models/Article.cs
namespace Backend.Models
{
    public class Article
    {
        public int Id { get; set; }
        public string? Authors { get; set; }
        public int? Year { get; set; }
        public string? Title { get; set; }
        public string? Journal { get; set; }
        public int? Issue { get; set; }
        public string? Place { get; set; }
        public string? Doi { get; set;}
        public string? Genre { get; set; }
        public string? Pdf { get; set; }

    }
}

File: frontend/src/app/models/article.ts
import { Genre } from "./genre"

export class Article{

    id:number = 0
    authors: string = ""
    year: number = 0
    title: string = ""
    journal: string = ""
    issue: number = 0
    place: string = ""
    doi: string = ""
    genreId: number = 0
    genre: Genre | undefined;
    pdf: string = ""
    pdfFile: any = ""

}